src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,122 @@
+package parser;
+
+import commands.*;
```

Comment:
Perhaps you can consider explicitly listing the imported classes for the commands package?